src/screens/GovernanceCall/GovernanceCallBanner.js

```diff
@@ -0,0 +1,191 @@
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+import * as React from 'react';
+import { Linking } from 'react-native';
+import styled from 'styled-components/native';
+import { useTranslationWithPrefix } from 'translations/translate';
+import {
+  addHours,
+  differenceInMinutes,
+  differenceInHours,
+  differenceInDays,
+  isAfter,
+  isBefore,
+} from 'date-fns';
+
+// Components
+import Image from 'components/Image';
+import Text from 'components/core/Text';
+
+// Utils
+import { fontStyles, spacing } from 'utils/variables';
+import { reportErrorLog } from 'utils/common';
+import { mapFromDocumentDataToString } from 'utils/prismic';
+import { formatDate } from 'utils/date';
+
+// Services
+import * as Prismic from 'services/prismic';
+
+// assets
+const smartWalletIcon = require('assets/icons/smart-wallet-migrate.png');
+
+const TYPE_GOVERNANCE_CALL = 'governance-calls';
+const DATE_FORMAT = 'dd.MM';
+const TIME_WITH_TIMEZONE_FORMAT = 'HH:mm OOO';
+
+const GovernanceCallBanner = () => {
+  const { t } = useTranslationWithPrefix('governanceCall');
+  const [title, setTitle] = React.useState(t('banner.title'));
+  const [description, setDescription] = React.useState('');
+  const [scheduledCallTime, setScheduledCallTime] = React.useState(0);
+  const [governanceCallLink, setGovernanceCallLink] = React.useState('');
+  const [showPermanently, setShowPermanently] = React.useState(false);
+
+  React.useEffect(() => {
+    async function fetchGovernanceCallData() {
+      try {
+        const data = await Prismic.queryDocumentsByType(TYPE_GOVERNANCE_CALL);
+        data?.results?.map((governanceCallData) => {
+          const bannerTitleContent = [];
+          mapFromDocumentDataToString(governanceCallData?.data?.title, bannerTitleContent);
+          setTitle(bannerTitleContent);
+          const bannerDescriptionContent = [];
+          mapFromDocumentDataToString(governanceCallData?.data?.description, bannerDescriptionContent);
+          setDescription(bannerDescriptionContent);
+          /* eslint-disable camelcase */
+          setScheduledCallTime(governanceCallData?.data?.scheduled_for);
+          setGovernanceCallLink(governanceCallData?.data?.link?.url);
+          setShowPermanently(governanceCallData?.data?.show_permanently);
+        });
+      } catch (error) {
+        reportErrorLog('Prismic content fetch failed', { error });
+      }
+    }
+    fetchGovernanceCallData();
+  }, [scheduledCallTime]);
+
+  const eventEndTime = addHours(new Date(scheduledCallTime), 1);
+  const formattedDate = formatDate(new Date(scheduledCallTime), DATE_FORMAT);
+  const formattedTimeWithTimezone = formatDate(new Date(scheduledCallTime), TIME_WITH_TIMEZONE_FORMAT);
+  const getTotalDays = differenceInDays(new Date(scheduledCallTime), Date.now());
+  const getTotalHours = differenceInHours(new Date(scheduledCallTime), Date.now());
+  const getTotalMinutes = differenceInMinutes(new Date(scheduledCallTime), Date.now());
```

Comment:
We've created 6 instances of ```new Date(scheduledCallTime)``` here when we could just define it once, and reference that one definition...

We could also do the same with ```Date.now()``` which is called three times...

Reply:
Done.